packages/formation/sass/modules/_m-form-elements.scss

```diff
@@ -179,13 +212,8 @@ button.form-button-disabled {
   }
 }
 
-.form-expanding-group {
-  margin-left: -2rem;
-  padding-left: 2rem;
-}
```

Comment:
This looks like a global style - is this related to the error message styling?

Reply:
@bkjohnson I reverted the form-expanding-group update because it is global style and does effect other components not in the scope of this work